### PR TITLE
Refactor rake tasks.

### PR DESCRIPTION
**Pull request by rotated8**
Add a task to upgrade users to admins, and one to only create the
default roles. Remove old default setup taks. Change file name for
populating users.

### CODE REVIEW

**Comment by rotated8** [comment]
Depends on VTUL/InstallScripts#42

File: lib/tasks/datarepo.rake [hunk 1]
```diff
@@ -68,4 +49,24 @@ namespace :datarepo do
       end
     end
   end
+
+  desc 'Upgrade users to admins.'
```
Comment (yinlinchen) [inline lib/tasks/datarepo.rake]: Upgrade user to which one? (admin or collection_admin)

File: lib/tasks/datarepo.rake [hunk 2]
```diff
@@ -1,27 +1,10 @@
...
-    default_password = "datarepo_password"
-    Rake::Task[:with_defaults].invoke(default_email, default_password)
-    if User.find_by_email(default_email).nil?
-      User.create(email: default_email, password: default_password, role_ids: [Role.where(name: 'admin').first.id])
-      puts "Default admin email is: #{default_email}; password is: #{default_password}. Please change the password."
-    else
-      puts "Default admin email is: #{default_email}"
+  desc 'Create default roles.'
+  task add_roles: :environment do
+    ['admin', 'collection_admin', 'collection_user'].each do |role_name|
```
Comment (yinlinchen) [inline lib/tasks/datarepo.rake]: What's the difference between admin and collection_admin?

File: lib/tasks/datarepo.rake (hunk 1 above)
Comment (rotated8) [inline lib/tasks/datarepo.rake]: Admin.

File: lib/tasks/datarepo.rake (hunk 2 above)
Comment (rotated8) [inline lib/tasks/datarepo.rake]: collection_admin and collection_user were added back when we came up with the roles. They do not do anything.

admin, on the other hand, has a meaning to Sufia, and our code.

**Pull request merged**